specs/merge/validator.md

```diff
@@ -75,28 +101,37 @@ def get_pow_block_at_total_difficulty(total_difficulty: uint256, pow_chain: Sequ
     return None
 
 
-def produce_execution_payload(state: BeaconState,
-                              parent_hash: Hash32,
-                              execution_engine: ExecutionEngine) -> ExecutionPayload:
-    timestamp = compute_timestamp_at_slot(state, state.slot)
-    randao_mix = get_randao_mix(state, get_current_epoch(state))
-    return execution_engine.assemble_block(parent_hash, timestamp, randao_mix)
-
-
-def get_execution_payload(state: BeaconState,
-                          transition_store: TransitionStore,
-                          execution_engine: ExecutionEngine,
-                          pow_chain: Sequence[PowBlock]) -> ExecutionPayload:
+def prepare_execution_payload(state: BeaconState,
+                              pow_chain: Sequence[PowBlock],
+                              fee_recipient: Bytes20,
```

Comment:
Consider defining a type for `ExecutionAddress`

Reply:
Done